features/src/main/scala/com/salesforce/op/stages/OpPipelineStages.scala

```diff
@@ -165,6 +159,11 @@ trait OpPipelineStageBase extends OpPipelineStageParams with MLWritable {
 
   final override def write: MLWriter = new OpPipelineStageWriter(this)
 
+
+  /**
+   * Function to be called on setInput
+   */
+  override protected def onSetInput(): Unit = {}
```

Comment:
@tovbinm any idea why I had to override the method here to make all the tests using the pipeline stage spec pass?